src/tools/pub/pubspec.md

```diff
@@ -181,6 +189,24 @@ helps users understand where your package is coming from. If nothing else, you
 can always use the URL where you host the source code, such as
 [GitHub](https://github.com).
 
+### Repository
+
+The `repository` property should be a URL pointing to the repository for the
+package. This is often the URL for a github repository:
+`https://github.com/<user>/<repository>`.
+For [hosted packages](/tools/pub/dependencies#hosted-packages),
+this URL will be displayed on package's page.
+
+### Issue Tracker
+
+The `issue_tracker` property should be a URL pointing to the issue tracker for
```

Comment:
How about:

Help your package's users find and file bugs in the right place by putting a URL in the `issue_tracker` field. The pub.dartlang.org site attempts to display a link to each package's issue tracker, using the value of this field. If `issue_tracker` is missing but [`repository`](#repository) is present and points to GitHub, then pub.dartlang.org uses the default issue tracker (`https://github.com/<user>/<repository>/issues`).